src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -763,6 +830,9 @@ class ApplicationManagerImpl
 
   void SendMessageToHMI(const commands::MessageSharedPtr message) OVERRIDE;
 
+  void RemoveHMIFakeParameters(
```

Comment:
what are fake parameters of HMI?

Reply:
@jacobkeeler it is nit unused parameters, it is parameters that are not present in API.

Reply:
I think `RemoveInvalidParameters` would be a better name than using `Fake`

Reply:
@JackLivio "Remove fake parameters" is official naming of SDL feature, I would prefer avoid mismatch of naming for better tractability

Reply:
@JackLivio "Remove fake parameters" is official naming of SDL feature, I would prefer avoid mismatch of naming for better tractability